### ISSUE TITLE
Add a way to create a new user account in single user mode

### ISSUE THREAD

**Issue by Lertsenem** [tootsuite/mastodon#833]
Right now the only way to create a new user with a single user instance is to directly request the database or to open the registrations for a while, and both ways aren't ideal.

A task, such as the make_admin one, could be a relatively easy (I guess, I'm not ruby literate) solution.

An admin command in the settings would also be could, but would probably need a bit more work.

**Comment by Gargron** [comment]
Solved by #1482

**Comment by Sieboldianus** [comment]
This was really difficult to find and this issue is shown first on google results, so I am adding it here. If your setup is docker:
```
docker compose exec web /bin/sh
tootctl accounts create sandy \
  --email mail@sandy.com \
  --confirmed \
  --role Owner
```
(with `sandy` as a sample admin user)

A random password will be returned on the shell.
